src/main/java/core/basesyntax/service/readandwrite/ReportWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.readandwrite;
+
+import core.basesyntax.service.parser.ReportServiceImpl;
+import core.basesyntax.service.parser.ReportService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static final String EXCEPTION_MESSAGE = "Can't write report in file ";
+    private ReportService reportService = new ReportServiceImpl();
+
+    @Override
+    public boolean writeReport(String pathToFile, String content) {
+        try {
+            Files.writeString(Path.of(pathToFile), reportService.generateReport());
```

Comment:
```suggestion
            Files.writeString(Path.of(pathToFile), content);
```
it's not enoygh to add content to method arguments, make this class responsible only for writing and pass ready content to the method

Reply:
done